### ISSUE TITLE
gen.py: SVG to PNG conversion uses deprecated Rsvg.Handle.close

### ISSUE THREAD

**Issue by BlueManCZ** [numixproject/numix-core#5728]
Works fine, but I'm getting this warning.

```
Generating Linux theme...
./gen.py:102: DeprecationWarning: Rsvg.Handle.close is deprecated
  svg.close()
Done!
```

https://developer.gnome.org/rsvg/stable/RsvgHandle.html#rsvg-handle-close

**Comment by palob** [comment]
What exactly causes this warning?

**Comment by Foggalong** [comment]
Yeah noticed tat too the other day, had a half hearted look through the `rsvg` APIs to see what they'd replaced it with and couldn't see anything obvious :sweat_smile:

**Comment by founderio** [comment]
This section of the docs might be helpful here: https://lazka.github.io/pgi-docs/Rsvg-2.0/classes/Handle.html#Rsvg.Handle.new

**Comment by Foggalong** [comment]
I did some script work in #4780 and was thinking of tackling this then, but reading these docs it sounds like it's the whole method of reading SVGs that's changed rather than just `.close`. @bilelmoussaoui I don't suppose this is an issue you've ran into with hardcode-tray?

**Comment by BlueManCZ** [comment]
I also noticed this. It seemed like a simple fix, but then I started experimenting with a code and it looks like most of the function has to be rewritten to fully avoid deprecation.

**Comment by palob** [comment]
I removed the SVG to PNG conversion part a while back in https://github.com/numixproject/numix-core/commit/20ba6d5963c8d0f8f4759b42049c697b70c941cd from the linux theme generation when the Unity 7 DE had become out of widespread use.
